components/server/ee/src/user/stripe-service.ts

```diff
@@ -61,7 +61,9 @@ export class StripeService {
             email: User.getPrimaryEmail(user),
             name: User.getName(user),
             metadata: {
+                // userId is deprecated, use attributionId where possible
                 userId: user.id,
+                attributionId: `user:${user.id}`,
```

Comment:
Nit: This format may change in the future. Please use the common `AttributionId.render` helper:

```suggestion
                attributionId: AttributionId.render({ kind: "user", userId: user.id }),
```